storage/storagepb/kv.proto

```diff
@@ -14,11 +14,14 @@ message KeyValue {
   int64 create_revision = 2;
   // mod_revision is the last modified revision of the key.
   int64 mod_revision = 3;
+  // tombstone_revision is the tombstone revision of the key.
+  // If the key is not tombstoned, its revision is set to zero.
+  int64 tombstone_revision = 4;
```

Comment:
a rev could show the detailed revision this key is deleted. So users could know when this key is deleted. It plays a similar functionality as mod_revision IMO.